package.json

```diff
@@ -51,7 +51,8 @@
     "test:visual:update": "yarn run build:storybook-visual && yarn loki update",
     "test:visual": "yarn loki test",
     "prepare": "husky install",
-    "size": "yarn run build && size-limit"
+    "size": "yarn run build && size-limit",
+    "tokens:update": "./scripts/figma-synchronise-tokens.sh https://raw.githubusercontent.com/scaleway/design-tokens/main/tokens.json && yarn run format"
```

Comment:
Better to run them outside in the action


```yarn run tokens:update && yarn run format```

Reply:
@chambo-e That's what I tough first but then I tough that anyone who will run `yarn run tokens:update` locally and won't have file correctly formatted will probably struggle to understand. I found it easier to just have one command that does it all, what do you think ?

Reply:
ok for me :)